### ISSUE TITLE
Storing access_token in UserDefaults

### ISSUE THREAD

**Issue by NikitaAtNimble** [galavebalaji/travelbooks#26]
Since access_token is sensitive data you should store it in more secure place rather than `UserDefaults`, for example in `Keychain`.

**Comment by galavebalaji** [comment]
Yes I had thought of it to store in the keychain or using some third party secure library but I considered not in scope and simply saved into ```UserDefaults```.
